src/components/layout/bottom-navigation.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import { BottomNavigation, Icon, Page } from "zmp-ui";
+
+const BottomNavigationComponent = (props) => {
+  const [activeTab, setActiveTab] = useState("chat");
+  const { title } = props;
```

Comment:
I think you should move "props" to the top of useState to make it easier to manage

Reply:
updated